File: API/Arosaje/ModelViews/Plant.cs
using System;
using System.Collections.Generic;

namespace Arosaje.ModelViews;

public partial class Plant
{
    public string? Id { get; set; }

    public string? IdUser { get; set; }

    public string? Name { get; set; }

    public DateTime? BeginAt { get; set; }

    public DateTime? EndAt { get; set; }

    public string? Description { get; set; }

    public byte[]? Picture { get; set; }

    public double Latitude { get; set; }

    public double Longitude { get; set; }
}

File: App/lib/core/data/entities/plant/plant.dart
import 'package:freezed_annotation/freezed_annotation.dart';

part 'plant.freezed.dart';
part 'plant.g.dart';

@freezed
class Plant with _$Plant {
  const factory Plant({
    required int? id,
    required int? idUser,
    required String? name,
    required String? beginAt,
    required String? endAt,
    required String? description,
    required String? picture,
    required double? latitude,
    required double? longitude,
  }) = _Plant;

  factory Plant.fromJson(Map<String, dynamic> json) => _$PlantFromJson(json);
}
